Antidote/CallsManager.m

```diff
@@ -205,28 +214,46 @@ - (void)activeCallSpeakerButtonPressed:(ActiveCallViewController *)controller
 
 - (void)activeCallPauseButtonPressed:(ActiveCallViewController *)controller
 {
+    NSError *error;
     if (controller.pauseSelected) {
-        if ([self.manager sendCallControl:OCTToxAVCallControlResume toCall:self.currentCall error:nil]) {
+        if ([self.manager sendCallControl:OCTToxAVCallControlResume toCall:self.currentCall error:&error]) {
             controller.pauseSelected = NO;
         }
+        else {
+            DDLogWarn(@"%@, activeCallPauseButtonPressed:OCTToxAVCallControlResume error: %@", self, error.localizedFailureReason);
+        }
     }
     else {
         if ([self.manager sendCallControl:OCTToxAVCallControlPause toCall:self.currentCall error:nil]) {
```

Comment:
Here you missed `&error` parameter. To avoid such mistakes it's better to avoid code duplication. :-) You can refactor this method like this:

```
- (void)activeCallPauseButtonPressed:(ActiveCallViewController *)controller
{
    NSError *error;
    OCTToxAVCallControl control = controller.pauseSelected ? OCTToxAVCallControlResume : OCTToxAVCallControlPause;

    BOOL result = [self.manager sendCallControl:control toCall:self.currentCall error:&error];

    if (result) {
        controller.pauseSelected = ! controller.pauseSelected;
    }
    else {
        DDLogWarn(@"%@, activeCallPauseButtonPressed:OCTToxAVCallControlPause error: %@", self, error);
    }
}
```